### DISCUSSION TITLE
Is GitHub Copilot good for ADHD

### DISCUSSION THREAD

**Discussion by ebertsch** [community/community#8175] [Copilot Conversations]
I have ADHD and I am curious if any users of Copilot have noticed an increase in their productivity since starting to use copilot. I have a hunch that Copilot will be life changing to those of us who are neurodivergent.

**Comment by GSPP** [comment]
Can you explain more about how it helped you? Like, how specifically helped you as an ADHD person?

**Reply by SmallLemur** [reply]
@ebertsch this is an interesting point, thank you for bringing it up, could you elaborate your thought on that? i would like to understand better :)

**Reply by ebertsch** [reply]
There are have been days in my past when I can struggle to write code. I know what I want to write but just cannot focus enough to write said code. In this scenario, if I code use copilot to generate the code and then just verify and modify it that would turn an unproductive day around.

My ideal bad day workflow instead of just staring at the screen
1) Create/open file
2) Use dictation to create my sentence of intent. (sometimes even typing is hard) "Read settings from a file"
3) Run Copilot on my sentence
4) Review and modify code

**Comment by uxkjaer** [comment]
I've got adhd and while I like your proposal I don't struggle in exactly the same way. But adhd is a spectrum right. However I do find Copilot really helpful because I very often make small stupid syntax mistakes or similar and Copilot helps me with these things. Just thought I'd elaborate on where I see a benefit.
